Solutions/Flare/Analytic Rules/FlareCredentialLeaks.yaml

```diff
@@ -0,0 +1,25 @@
+id: 467cbe7e-e6d4-4f4e-8f44-84dd01932c32
```

Comment:
Please generate a new Id. the validations also seems to be having some errors here.
Error message:
Template Id: 467cbe7e-e6d4-4f4e-8f44-84dd01932c32 is not valid in Line: 1 col: 1

Reply:
Done